src/annotator/bucket-bar.tsx

```diff
@@ -41,6 +41,10 @@ export class BucketBar implements Destroyable {
       // host pages using a custom bucket bar container don't need to hardcode
       // assumptions about its width.
       width: '100%',
+
+      // Allow pointer events to go through this container to page elements
+      // (eg. scroll bar thumbs) which are behind it.
+      pointerEvents: 'none',
```

Comment:
This was previously set in the whole `sidebar-edge` container via `pointer-events-none`, but it was meant to affect the buckets only, so moving it here to allow toolbar buttons to be "actionable".

Reply:
This doesn't work unfortunately because the `sidebar-edge` parent of this element no longer has `pointer-events-none`.

On http://localhost:3000/document/whitespace for example, if you resize the page such that a small amount of text (end of the word "whitespace") is visible behind the bucket bar:

<img width="1195" alt="bucket-bar-background" src="https://github.com/user-attachments/assets/029468be-a51c-4c75-889d-f6866612d796" />

Then on `main`, you can select that text behind the bucket bar. On this branch you can't.

Reply:
Hmm, good catch.

Reply:
Actually, I'm not being able to reproduce this. Maybe it's a Safari thing? 🤔

Reply:
I can reproduce in Chrome as well. I can't think of a reason why the behavior would be browser-specific at all. The problem is that the light grey background element in the screenshot above no longer has `pointer-events: none` on it, and hence you can't click through it.

Reply:
oh, wait. I was testing this incorrectly, sorry. I did not realize the sidebar had to be collapsed to test it.

I can in fact reproduce it now.

Reply:
I left the `pointer-events-none` class in the container, as it was before, and explicitly overwritten it in the toolbar via `pointer-events-auto`.